smart-contracts/osmosis/contracts/cl-vault/tests/test-tube/setup.rs

```diff
@@ -25,7 +25,7 @@ pub const ADMIN_BALANCE_AMOUNT: u128 = 100_000_000_000_000_000_000_000_000_000u1
 pub const PERFORMANCE_FEE_DEFAULT: u64 = 20;
 
 const TOKENS_PROVIDED_AMOUNT_HIGH: &str = "100000000000000000000";
-pub const SPREAD_FACTOR_HIGH: &str = "0.1";
+pub const SPREAD_FACTOR_HIGH: f64 = 0.1;
```

Comment:
Because i needed to make math ops with it in the test tube test autocompound.rs, and i preferred to have an origin f64. So we convert it to string at the time of instantiating contracts rather than in our test logic.